### PR TITLE
Fix link to HTTP/2 rfcs

### PR DESCRIPTION
**Pull request by yeynii**
#### Summary
invaild link

#### Supporting details
https://http2.github.io/http2-spec/#rfc.section.8.1.2.4 -> https://httpwg.org/specs/rfc7540.html#rfc.section.8.1.2.4
https://http2.github.io/http2-spec/#HttpResponse -> https://httpwg.org/specs/rfc7540.html#HttpResponse

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/xmlhttprequest/index.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ _This interface also inherits properties of {{domxref("XMLHttpRequestEventTarget
 - {{domxref("XMLHttpRequest.statusText")}} {{readonlyinline}}
   - : Returns a {{domxref("DOMString")}} containing the response string returned by the HTTP server. Unlike {{domxref("XMLHttpRequest.status")}}, this includes the entire text of the response message ("`200 OK`", for example).
 
-    > **Note:** According to the HTTP/2 specification ([8.1.2.4](https://http2.github.io/http2-spec/#rfc.section.8.1.2.4) [Response Pseudo-Header Fields](https://http2.github.io/http2-spec/#HttpResponse)), HTTP/2 does not define a way to carry the version or reason phrase that is included in an HTTP/1.1 status line.
+    > **Note:** According to the HTTP/2 specification ([8.1.2.4](https://httpwg.org/specs/rfc7540.html#rfc.section.8.1.2.4) [Response Pseudo-Header Fields](https://httpwg.org/specs/rfc7540.html#HttpResponse)), HTTP/2 does not define a way to carry the version or reason phrase that is included in an HTTP/1.1 status line.
```
Comment (nschonni) [inline files/en-us/web/api/xmlhttprequest/index.md:56]: Using the macro will point to https://datatracker.ietf.org/doc/html/rfc7540#section-8.1.2.4
```suggestion
    > **Note:** According to the HTTP/2 specification {{RFC(7230, "Response Pseudo-Header Fields", "8.1.2.4")}}, HTTP/2 does not define a way to carry the version or reason phrase that is included in an HTTP/1.1 status line.
```

**Review by teoli2003 — APPROVED** [review]
Thanks a lot!

File: files/en-us/web/api/xmlhttprequest/index.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ _This interface also inherits properties of {{domxref("XMLHttpRequestEventTarget
 - {{domxref("XMLHttpRequest.statusText")}} {{readonlyinline}}
   - : Returns a {{domxref("DOMString")}} containing the response string returned by the HTTP server. Unlike {{domxref("XMLHttpRequest.status")}}, this includes the entire text of the response message ("`200 OK`", for example).
 
-    > **Note:** According to the HTTP/2 specification ([8.1.2.4](https://http2.github.io/http2-spec/#rfc.section.8.1.2.4) [Response Pseudo-Header Fields](https://http2.github.io/http2-spec/#HttpResponse)), HTTP/2 does not define a way to carry the version or reason phrase that is included in an HTTP/1.1 status line.
+    > **Note:** According to the HTTP/2 specification {{RFC(7230, "Response Pseudo-Header Fields", "8.1.2.4")}}, HTTP/2 does not define a way to carry the version or reason phrase that is included in an HTTP/1.1 status line.
```
Comment (yeynii) [inline files/en-us/web/api/xmlhttprequest/index.md:56]:
```suggestion
    > **Note:** According to the HTTP/2 specification {{RFC(7540, "Response Pseudo-Header Fields", "8.1.2.4")}}, HTTP/2 does not define a way to carry the version or reason phrase that is included in an HTTP/1.1 status line.
```

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**